### DISCUSSION TITLE
Fun with Versions

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1431] [Show and tell]
So, if you're ever bored or want an example of multiple versions in PubPub, enjoy: https://www.sareview.org/pub/sar5texascovidtimeline/release/42

Note the 42.

**Comment by gabestein** [comment]
![DeepThought](https://user-images.githubusercontent.com/639110/119991065-0f254600-bf97-11eb-8e76-9369b0baae3b.png)
